docs/docs/installation/guided-install/index.md

```diff
@@ -4,6 +4,18 @@
 
 {% include 'snippets/docker-compose-prerequisites.md' %}
 
+## Quickstart
```

Comment:
Do we need a quickstart for this setup? The "Installation Steps" section is quite similar and might overlap with the "Quickstart" section. It was useful when it was part of the Index page(which is being deleted in this PR)

Reply:
I'll probably delete the quickstart sections, I didn't want to lose the content from the page that I deleted so I put them here temporarily.